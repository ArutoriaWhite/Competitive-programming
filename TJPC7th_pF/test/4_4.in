7404504 998
2252 1
10942 0
11034 0
16563 1
18404 1
23173 1
37260 0
44812 1
47020 0
52009 0
60832 1
61188 1
64858 0
69118 1
87683 0
93134 1
124746 0
134599 0
146895 0
158023 0
160044 0
162799 0
172624 0
179190 1
181338 1
190730 0
204790 1
225245 1
225787 0
253820 1
264031 0
277075 0
281657 0
284199 1
284630 0
289332 0
290191 1
295464 1
300822 0
303289 0
312855 1
313810 0
341138 1
364456 0
368018 0
376414 1
378069 0
396176 0
404026 1
421386 1
442162 1
449501 1
454619 0
460444 0
474525 1
488894 1
499054 0
516238 1
522274 1
527083 0
531254 1
553814 0
555214 0
557619 1
564124 1
564815 0
580641 0
584583 1
602982 0
623684 1
623864 0
631073 0
631978 1
639258 0
642079 0
647395 0
655302 0
656529 0
658191 0
674261 0
675592 1
693895 0
695293 1
696021 1
708376 0
709159 1
715809 0
724759 0
726977 1
739689 1
742148 0
774174 1
780669 1
782165 1
785057 1
789592 0
790482 1
794159 1
794507 0
803818 0
805985 0
808438 1
808561 1
826023 0
838869 1
846537 0
850546 0
862794 0
863368 1
864145 0
867740 0
878326 0
882210 0
898446 0
912076 0
913556 0
919666 1
922093 0
928787 0
939558 0
949999 1
950057 1
955954 1
962296 1
965703 1
971182 0
972483 0
973846 0
975395 1
977268 1
980273 1
981985 1
998624 1
1015933 0
1019020 1
1041196 0
1042416 0
1061217 0
1067055 1
1073113 1
1080908 0
1093117 1
1097461 0
1104937 0
1112817 1
1120382 1
1125483 1
1131906 1
1136887 1
1139058 0
1141369 0
1170286 1
1170638 0
1171544 1
1174976 0
1212906 0
1214137 1
1226202 1
1246029 0
1254783 1
1255855 1
1263354 1
1268218 0
1274187 0
1275010 1
1289814 1
1310943 1
1318921 1
1326214 0
1326576 0
1339157 0
1343216 1
1354855 0
1358830 1
1362608 1
1368517 0
1369287 0
1401094 1
1405843 1
1408600 1
1409947 1
1413089 0
1427200 0
1439059 0
1451816 1
1457528 0
1473641 0
1482443 0
1488610 1
1495559 0
1500239 1
1516817 0
1523726 1
1526347 0
1535447 1
1539679 0
1552823 1
1553019 1
1562699 1
1564679 1
1566155 1
1571341 0
1573125 1
1597714 1
1600423 0
1607784 0
1619993 1
1622025 1
1630381 0
1634073 1
1651688 1
1652249 1
1674827 1
1681262 1
1681766 1
1692215 0
1697896 0
1701433 0
1705085 0
1713342 1
1715141 0
1727619 0
1731014 1
1739364 0
1747232 0
1750774 0
1776697 0
1786989 0
1788251 1
1797237 1
1828383 0
1831475 0
1844654 0
1845117 0
1847580 0
1857696 1
1860869 1
1867254 0
1879758 1
1883634 1
1885121 0
1890111 1
1895795 1
1896348 0
1899965 1
1918506 0
1920516 0
1921402 1
1921938 1
1926907 1
1928551 0
1937959 1
1938967 0
1941325 0
1944878 0
1969522 0
1979338 0
1985057 1
2001396 0
2007862 1
2014406 1
2019489 0
2024348 0
2038339 0
2042567 1
2050964 1
2052897 1
2087499 0
2099172 0
2103636 1
2103969 1
2114099 0
2116775 0
2120212 1
2122382 1
2123473 0
2125021 0
2127534 0
2138816 1
2147756 1
2154454 0
2159376 1
2161028 1
2170426 0
2171418 1
2174379 1
2182197 0
2187479 0
2188267 0
2189394 1
2200515 1
2203991 1
2214387 0
2223826 0
2238877 1
2253305 0
2263459 0
2264673 0
2266238 1
2280800 0
2285001 1
2293799 0
2300132 1
2306128 0
2309468 1
2314416 0
2319932 1
2322712 0
2326262 1
2331009 1
2341487 0
2351199 1
2352595 1
2358291 0
2364101 1
2376938 1
2385263 1
2395034 1
2396615 1
2397040 0
2410683 0
2413054 1
2432526 0
2438207 1
2450315 0
2450830 0
2458523 1
2462684 0
2465613 1
2467244 1
2467983 0
2477534 0
2492162 1
2500533 1
2500827 1
2503757 0
2504109 0
2505859 0
2506626 0
2515130 0
2522371 1
2522857 1
2527066 1
2533110 0
2541524 1
2541750 1
2543377 0
2551492 1
2565890 0
2566096 0
2575122 1
2584284 0
2587930 0
2594006 0
2617214 0
2618549 0
2627241 0
2633663 1
2638057 0
2639674 0
2665290 1
2669176 0
2678220 0
2680025 1
2708605 0
2712772 0
2722659 1
2737881 1
2745842 0
2756830 1
2768598 0
2787918 0
2797057 1
2805339 0
2807779 1
2812505 0
2819488 1
2830978 1
2834436 1
2840677 0
2844492 0
2848597 1
2850343 1
2868319 0
2874340 1
2877425 1
2879018 1
2879726 0
2880088 0
2880241 0
2890083 1
2893289 1
2902579 1
2906965 0
2910101 1
2915056 0
2927907 1
2931544 0
2947230 0
2947925 1
2953140 1
2972283 1
2977271 0
3001273 0
3008874 0
3010959 0
3018538 1
3022715 1
3023904 1
3025861 1
3026399 1
3027175 1
3039043 0
3040396 0
3044435 0
3044480 1
3052205 0
3054494 1
3062287 1
3064709 0
3069607 1
3070227 0
3073114 0
3078520 1
3078976 0
3082754 1
3085383 1
3086059 1
3098021 1
3114091 0
3115373 1
3123132 0
3135387 1
3137723 1
3146224 1
3156737 0
3159955 1
3163007 1
3163306 1
3163897 0
3167477 1
3167897 1
3168248 0
3185607 1
3219228 1
3223614 0
3229382 0
3244821 0
3257732 0
3259062 1
3273366 0
3278953 0
3294699 1
3299530 1
3309034 0
3313658 1
3322212 1
3329281 1
3330478 0
3331819 1
3332757 1
3336479 0
3344383 0
3349349 0
3355587 0
3358822 0
3364958 0
3365700 0
3366479 0
3368434 1
3369424 0
3369636 0
3371535 1
3386071 0
3395662 0
3396233 0
3399322 0
3438990 1
3449223 0
3454813 1
3467303 0
3468455 1
3483327 0
3488277 0
3495124 1
3511534 1
3513396 0
3514319 1
3533377 0
3539520 1
3540979 1
3550125 1
3578802 1
3601263 1
3604336 0
3622319 0
3625661 0
3628334 0
3658473 0
3660732 0
3665469 0
3678079 1
3683145 0
3684979 1
3687000 1
3691778 0
3692928 1
3694527 0
3706033 1
3707278 1
3714868 0
3715123 0
3716909 0
3717849 0
3718336 0
3726735 1
3729818 0
3743876 1
3752013 0
3756554 1
3757184 0
3763337 0
3764825 1
3768344 1
3776985 0
3777418 0
3785703 0
3801896 0
3812161 0
3815135 1
3818011 1
3829889 0
3830773 1
3831594 0
3834436 0
3847323 0
3876102 0
3882245 1
3888665 1
3889436 1
3891985 1
3899058 1
3907882 0
3922934 0
3931747 1
3936272 0
3937496 0
3944148 0
3968089 0
3968294 0
3978415 1
3991482 0
4002064 0
4025436 1
4033129 0
4038796 0
4038824 1
4044397 0
4048405 1
4069310 1
4076706 1
4085490 0
4097059 1
4107900 0
4108568 1
4111655 1
4112054 1
4119162 1
4130086 1
4136686 0
4138720 1
4150017 0
4154625 1
4166250 0
4168448 0
4168949 0
4172800 1
4176232 0
4176837 0
4188046 0
4199481 0
4222150 1
4223379 0
4223538 1
4232883 1
4235352 0
4246427 1
4247858 1
4260461 1
4271476 0
4279752 0
4289178 0
4292668 1
4316237 1
4330030 1
4351410 1
4357024 0
4358321 1
4368524 0
4373275 0
4373498 0
4378288 0
4385920 0
4386879 1
4389016 0
4395709 0
4402674 0
4404376 1
4404400 1
4406616 0
4416984 1
4431384 1
4433125 0
4445587 0
4455390 1
4457722 0
4462153 1
4483173 0
4483293 0
4489919 0
4490119 1
4490456 0
4499626 1
4499762 0
4505368 1
4524848 1
4544070 0
4545386 0
4546481 0
4557788 0
4561103 0
4575791 1
4584534 1
4608938 1
4630517 0
4636186 1
4636606 1
4639743 0
4658398 0
4671803 0
4672434 1
4676466 0
4706580 1
4719761 1
4722232 1
4723082 1
4726516 0
4726677 1
4730299 0
4749148 0
4757756 0
4760878 0
4766507 0
4780187 1
4785708 0
4790090 0
4790371 1
4794380 1
4830717 0
4837167 1
4846099 1
4853594 1
4863577 1
4864905 0
4865730 1
4902805 1
4907456 1
4913350 1
4922762 0
4927951 0
4928867 1
4931453 0
4937414 0
4939630 0
4941142 1
4958560 1
4960879 1
4966991 1
4971002 1
4971071 0
4975833 1
4976905 1
4992828 0
5003309 0
5011424 1
5015726 0
5026658 1
5030112 1
5030986 0
5032264 1
5036316 1
5056169 1
5059074 1
5105363 0
5122801 0
5144849 1
5150088 0
5151095 1
5153173 0
5166739 0
5180771 1
5186227 0
5204788 0
5214491 0
5223378 1
5231479 1
5245991 0
5250287 0
5253761 1
5260700 1
5263111 1
5264168 0
5273660 0
5274834 1
5283000 1
5289345 1
5289817 0
5293640 0
5298088 0
5303111 1
5314143 1
5371627 1
5394617 0
5399346 1
5406020 1
5425659 1
5430975 0
5431019 0
5439191 0
5441140 0
5442673 0
5444820 0
5449045 1
5453575 1
5471874 1
5482221 0
5499616 0
5504645 1
5515899 0
5526390 1
5534763 0
5537117 1
5538062 0
5548854 1
5552727 0
5561203 0
5565463 1
5577660 0
5579788 0
5585431 0
5592771 1
5603836 1
5616995 1
5644827 1
5645025 0
5654637 0
5658995 0
5670679 1
5670762 1
5671185 0
5673118 1
5675163 0
5681777 1
5682018 0
5682473 1
5686956 1
5692093 0
5702751 0
5706093 1
5733929 1
5737405 0
5743363 1
5746479 0
5753559 1
5754060 1
5768886 1
5769525 1
5780811 0
5781311 0
5783423 0
5795416 1
5807155 1
5810330 0
5819234 0
5828561 0
5829410 0
5835083 0
5839282 1
5844181 0
5847801 1
5859148 1
5859481 0
5868108 0
5872056 0
5881737 0
5884966 1
5887512 0
5890817 1
5894858 1
5898879 0
5905376 0
5908110 0
5921092 1
5928370 0
5929614 1
5935728 0
5938268 0
5944902 1
5952010 1
5956490 0
5966407 1
5969983 1
5977101 1
5980990 0
5996574 1
6001446 1
6007289 0
6009709 1
6015019 0
6028595 1
6060372 1
6062918 0
6071561 1
6075878 1
6090790 0
6115717 0
6116877 0
6121330 1
6126071 0
6132565 0
6133475 1
6135957 1
6138781 1
6143600 1
6146164 0
6161065 1
6164495 0
6169183 0
6183612 1
6187678 1
6187799 1
6194088 0
6209380 1
6227917 0
6229698 1
6231453 0
6266753 0
6279047 0
6279740 1
6286937 0
6287102 1
6287662 1
6295848 0
6307539 1
6318968 0
6321204 0
6326344 0
6331944 0
6348849 1
6356122 0
6373952 1
6375341 1
6379642 1
6395925 0
6396487 0
6398070 0
6403022 0
6403987 1
6408325 0
6411663 1
6424082 0
6429565 0
6435196 0
6436284 0
6439155 0
6439456 0
6447911 1
6449371 1
6462669 1
6463588 0
6474924 1
6487457 1
6516709 1
6531838 1
6538675 1
6541155 0
6550365 1
6562587 1
6565314 0
6565748 0
6565864 0
6572192 0
6582817 0
6585168 1
6610734 1
6619518 1
6626872 1
6648311 1
6658847 0
6666934 1
6673384 1
6679709 1
6683592 0
6698683 0
6705887 0
6734382 0
6754851 0
6757811 0
6762730 0
6779560 1
6790764 0
6800898 1
6811736 0
6813799 0
6833607 0
6833835 1
6873209 0
6876781 1
6886724 1
6887875 0
6890415 0
6896667 0
6902370 1
6909985 0
6912032 0
6924123 1
6939543 0
6942205 1
6943497 1
6944886 1
6945158 1
6959317 1
6970268 0
6983760 0
6984585 0
6987514 1
6997119 1
6998843 0
7001191 0
7004460 1
7013741 0
7019266 0
7035317 1
7039267 0
7043434 0
7043606 0
7044225 0
7053530 1
7056933 0
7067625 0
7079514 1
7092949 1
7095265 0
7108706 1
7115704 1
7116122 1
7117318 0
7117885 1
7119073 1
7121387 1
7123086 1
7124862 0
7131333 1
7132195 1
7139605 0
7141496 1
7151351 0
7156860 0
7160869 0
7191314 0
7203060 0
7208686 1
7209016 0
7212253 1
7218431 1
7225976 0
7246654 0
7259742 0
7270941 0
7271286 0
7280444 1
7285374 0
7295896 1
7298303 0
7302400 1
7302655 0
7319331 1
7329311 1
7329529 1
7356686 1
7369798 0
7373011 0
7379853 0
7385153 0
7385777 0
7391631 0
7394339 1
7394643 0
