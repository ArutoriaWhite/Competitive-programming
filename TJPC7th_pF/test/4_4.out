4575791 7353561
